### DISCUSSION TITLE
PDF thumbnail, Whatsapp style

### DISCUSSION THREAD

**Discussion by marques620** [quasarframework/quasar#12695] [General - Components / Directives / etc]
Hi there,

I'm trying to create a documents page and show all the documents as Whatsapp does,
![Untitled](https://user-images.githubusercontent.com/99146124/157108779-68eff6da-5f25-4917-be99-d0a8bf7bf294.png)

I've been looking for ways to do it but nothing actually worked, if someone has an idea, it would help A LOT,

Thanks.

**Comment by MilosPaunovic** [comment]
Are you looking for the card-like element or the way to show PDF thumbnail image?

**Comment by marques620** [comment]
Hey,

Both, I want to create a list of documents with the q-card element and
display a thumbnail of the PDF on it.

**Reply by MilosPaunovic** [reply]
Okay, have you started with something and got stuck... That's the way to get help, try to do it yourself first and if you have specific problems, ask about 'em and get directions.

**Comment by zutamacka** [comment]
Try this: 
https://github.com/quasarframework/quasar/discussions/10623

I just punched it in like this in a page to test it;
`<template> 
    <q-item>
      <q-pdfviewer type="html5" src="https://www.orimi.com/pdf-test.pdf"
    /></q-item>
</<template>`

It worked.

**Reply by CynthiaBlue** [reply]
Is q-pdfviewer a regular quasar tag? Is there a quasar page about that? I'm looking for ways to generate PDFs with quasar & electron.

**Reply by MilosPaunovic** [reply]
[https://qpdfviewer.netlify.app/](https://qpdfviewer.netlify.app/)

**Reply by CynthiaBlue** [reply]
It may not be working with electron, I'm waiting for an answer to my comment.    Any other suggestions? I need to get PDF working in my quasar/electron app. :p 

https://github.com/quasarframework/app-extension-qpdfviewer/issues/7
